website/views/pages/imagine/jamf-alternative.ejs

```diff
@@ -0,0 +1,74 @@
+<div id="jamf-alternative" v-cloak>
+
+  <div purpose="hero-background" class="container-lg d-flex justify-content-center">
+
+    <div purpose="hero-container" class="d-flex flex-column align-items-center">
+      <div purpose="hero-text">
+        <h4 class="mb-2">Device management built for IT, security approved</h4>
+        <h1>Fleet brings GitOps to MDM</h1>
+        <p>Bringh-your-own MDM. Enjoy enterprise-ready open-source MDM and leverage the best of DevOps and GitOps inside a full-featured Macbook MDM.</p>
+        <div purpose="button-row" class="d-flex flex-sm-row flex-column justify-content-center align-items-center">
+          <a purpose="cta-button" href="/try-fleet/register?tryitnow">Try Fleet today</a>
+          <a @click="clickOpenChatWidget()" purpose="animated-arrow-button-red">Talk to an expert</a>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div purpose="page-container" class="container-lg">
+    <div purpose="feature" class="d-flex flex-md-row flex-column-reverse justify-content-between mx-auto align-items-center">
+      <div class="d-flex flex-column">
+        <h3>Finally a transparent and open-source MDM.</h3>
+        <p>Don't waste money buying new Protection plans from your MDM provider.</p>
+      </div>
+      <div purpose="feature-image" class="ml-md-5">
+        <img alt="jamf-alternative feature image" src="https://my.feralgoblin.com/cass/300/300?text=jamf alternative+image+1">
```

Comment:
@DrewBakerfdm, this is still using a placeholder image. Do we have images we can use for this page?